src/pages/Lesson4.tsx

```diff
@@ -0,0 +1,116 @@
+import { FC } from 'react';
+import AppWrapper from '../components/AppWrapper';
+import FillArrow from '../components/FillArrow';
+import NavButtons from '../components/NavButtons';
+import Note from '../components/Note';
+import Terminal from '../components/Terminal';
+import { HeaderSections } from '../types/globalTypes';
+
+const code1 = `
+  * // = "get the value stored at this address" AKA dereferencing
+  `;
+const code2 = `
+  int* ptr = &x;
+  `;
+
+const code3 = `
+  int x = *ptr;
+  `;
+
+const Lesson4: FC = () => {
+  return (
+    <>
+      <AppWrapper section={HeaderSections.LESSON_4}>
+        <div className="page-wrapper">
+          <h1>Dereferencing Pointers</h1>
+          <div className="diagram">
+            <FillArrow text1="myPointer" text2="myVariable" />
+          </div>
+          <p>
+            Now you might be wondering, what good will knowing an address do?
+            And you&apos;re right - most of the time, we don&apos;t care about
+            the actual value of the address itself, we care about the value
+            stored at that address. So if we are just given an address, it would
+            be really useful if there were a way for us to find out what value
+            is stored there. And you guessed it, there is!
+          </p>
+          <label>Boxed Syntax Notation:</label>
```

Comment:
Sorry, should have caught this earlier on the Figma, but this label was just for indicating the design, not for inclusion, Can you remove this from this lesson as well as from Lesson 3? I'll write something up to quickly explain the boxed syntax.